cdap-docs/application-templates/source/etl/templates/validators/core.rst

```diff
@@ -0,0 +1,59 @@
+.. meta::
+:author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+================
+Validators: Core
+================
+
+.. rubric:: Description
+
+A System Validator that offers a set of function's that can be called from validator transform.
+
+.. rubric:: Use Case
+
+Often want to validate the input for certain data-type or format or check if they are valid date, credit card etc.
+Its useful to implement and aggregate these common functions and make it easily available.
```

Comment:
Users often want to validate the input for a certain data-type or format, or check if it is a valid date, credit card, etc.
It's useful to implement and aggregate these common functions and them easily available.